sdk/main/include/AccountCreateTransaction.h

```diff
@@ -278,53 +281,50 @@ class AccountCreateTransaction : public Transaction<AccountCreateTransaction>
 
   /**
    * The key that must sign each transfer out of the account. If mReceiverSignatureRequired is \c TRUE, then it must
-   * also sign any transfer into the account. Defaults to nullptr (uninitialized).
+   * also sign any transfer into the account.
    */
-  std::shared_ptr<PublicKey> mKey;
+  std::shared_ptr<PublicKey> mKey = nullptr;
 
   /**
-   * The initial amount to transfer into the new account. Defaults to 0 Hbar.
+   * The initial amount to transfer into the new account.
    */
   Hbar mInitialBalance = Hbar(0LL);
 
   /**
    * If \c TRUE, the new account's key must sign any transaction being deposited into it (in addition to all
-   * withdrawals). Defaults to \c FALSE.
+   * withdrawals).
    */
   bool mReceiverSignatureRequired = false;
 
   /**
    * A Hedera account is charged to extend its expiration date every renew period. If it doesn't have enough balance, it
-   * extends as long as possible. If the balance is zero when it expires, then the account is deleted. Defaults to 3
-   * months.
+   * extends as long as possible. If the balance is zero when it expires, then the account is deleted.
    */
-  std::chrono::duration<int64_t> mAutoRenewPeriod = std::chrono::months(3);
+  std::chrono::duration<double> mAutoRenewPeriod = std::chrono::months(3);
```

Comment:
The changing to a `double` template parameter type here doesn't impact the conversion to the protobuf object; those are cast to their respective types already. I changed everything to a `double` here because it allows for easier conversions between `std::chrono` typedefs than `duration<int64_t>` types do while still retaining a generic `duration` type.